Makefile

```diff
@@ -130,6 +149,30 @@ build: # Build REANA client and cluster components.
 	reana-dev git-submodule --update && \
 	reana-dev docker-build -b DEBUG=${DEBUG} $(addprefix --exclude-components , ${EXCLUDE_COMPONENTS})
 
+release-build:
+	@echo -e "\033[1;32m[$$(date +%Y-%m-%dT%H:%M:%S)]\033[1;33m reana:\033[0m\033[1m make release-build\033[0m"
+	source ${HOME}/.virtualenvs/${INSTANCE_NAME}/bin/activate && \
+	minikube docker-env --profile ${INSTANCE_NAME} > /dev/null && eval $$(minikube docker-env --profile ${INSTANCE_NAME}) && \
+	echo "Upgrading REANA to latest master ..." && \
+	reana-dev git-upgrade -c CLUSTER && \
+	echo "Cleaning components directories ..." && \
+	reana-dev git-clean -c CLUSTER && \
+	BUILT_IMAGES_FILE_PATH=$(shell echo /tmp/reana-images-`git describe --dirty`.txt) && \
+	echo "Building images with no cache ..." && \
+	reana-dev docker-build -c CLUSTER --no-cache \
+		$(foreach argument, ${BUILD_ARGUMENTS}, -b ${argument}) \
+		-u "${GITHUB_USER}" -t auto \
+		--output-component-versions $$BUILT_IMAGES_FILE_PATH \
+		$(addprefix --exclude-components , ${EXCLUDE_COMPONENTS}) && \
+	echo "Building reanahub/krb5 image" && \
+	REANA_KRB5_IMAGE_NAME=$(shell cd ../reana-job-controller && echo reanahub/krb5:latest) && \
```

Comment:
Why is it needed here `cd ../reana-job-controller`? You're doing it in the next step.

Reply:
This is actually a leftover, before I was building the tag with `git describe dirty` but we only point to latest, thanks!